src/data/staking-products.json

```
@@ -190,6 +190,33 @@
         "eventAction": "Clicked",
         "eventName": "Clicked Vouch + Dirk go to link"
       }
+    },
+    {
+      "name": "AVADO",
```

Curious if there would be opposition to making this title case consistent with the others?
```suggestion
      "name": "Avado",
```